src/main/java/com/weaponcharges/ChargedWeapon.java

```diff
@@ -623,6 +623,54 @@ periodic updates (periodic, empty):
 			ChargesMessage.matcherGroupChargeMessage("Your crystal legs has ([\\d,]+) charges remaining.", 1)
 		)
 	),
+	/* Serpentine Helmet:
```

Comment:
An explanation of the serp helm degradation mechanics, as well as the likely deficiencies of the current approach visavis ranged attacks would be nice here.

Reply:
Also the barrows armor timer sharing.

Reply:
Implemented in commit 169e8e8360554165fa4eead7377f82ae5817bfc3.